module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -95,16 +95,17 @@ public function getTranslatorLocale($default = 'en')
     /**
      * Translate a string (or string-castable object)
      *
-     * @param string|object|array $target  String to translate or an array of text
+     * @param string|object|array $target           String to translate or an array of text
      * domain and string to translate
-     * @param array               $tokens  Tokens to inject into the translated
-     * string
-     * @param string              $default Default value to use if no translation is
+     * @param array               $tokens           Tokens to inject into the translated string
+     * @param string              $default          Default value to use if no translation is
      * found (null for no default).
+     * @param bool                $messageFormatter Should we use an ICU message formatter instead
+     * of the default behavior?
      *
      * @return string
      */
-    public function translate($target, $tokens = [], $default = null)
+    public function translate($target, $tokens = [], $default = null, $messageFormatter = false)
```

Comment:
I was confused initially expecting an actual formatter object to be passed in here.  (Before reading the doc.)  Because 'false' as a default doesn't imply the actual variable is a boolean, since it's used as a null substitute all over.   Maybe rename to $useMessageFormatter?

Reply:
Agreed -- I changed it to $useIcuFormatter to be both shorter AND more specific. :-)